pkg/edition/java/proxy/session_backend_login.go

```diff
@@ -162,7 +164,7 @@ func createVelocityForwardingData(hmacSecret []byte, address string, profile *pr
 	}
 
 	// final
-	data := bytes.NewBuffer(make([]byte, mac.Size()+forwarded.Len()))
+	data := bytes.NewBuffer(make([]byte, 0, mac.Size()+forwarded.Len()))
```

Comment:
First instance of this was fixed above, but was still getting a lot of leading 0s because of this. Should *probably* change to `new(bytes.Buffer)` syntax, but 🤷